### DISCUSSION TITLE
[UNSTABLE] KOTH is completely unplayable

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14283] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It spawns an outpost as if it was outpost deathmatch. There is a beacon but good luck reaching it if you weren't the first to the sub.

### Reproduction steps

1. Start a game of KOTH
2. suffer

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/e268dc11-e863-4784-811f-63946b92d7d3)

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next Unstable build.
